config/typescript/modules.d.ts

```diff
@@ -5,3 +5,22 @@ declare module 'url' {
 
   export function parse(url: string): URLDetails;
 }
+
+declare module 'react-tree-walker' {
```

Comment:
Can we make a PR to add this to DefinitelyTyped, or this library itself (later)?

Reply:
https://github.com/Shopify/quilt/issues/250